README.md

```diff
@@ -60,8 +61,13 @@ Contributions are welcomed!
 * **Jeff Rothenberg** (January 2006), [Renewing _The Erl King_](http://archive.bampfa.berkeley.edu/about/ErlKingReport.pdf)
 * **Christoph Becker, Günther Kolar, Josef Küng, and Andreas Rauber** (December 2007), [Preserving Interactive Multimedia Art: A Case Study in Preservation Planning](https://publik.tuwien.ac.at/files/pub-inf_5204.pdf), _Asian Digital Libraries. Looking Back 10 Years and Forging New Frontiers: 10th International Conference on Asian Digital Libraries_. Proceedings of ICADL 2007, Hanoi, Vietnam, 10-13 December 2007, 257-266.
 * **ZKM | Center for Art and Media Karlsruhe** (2010-2012), [digital art conservation: 10 case studies](http://www.digitalartconservation.org/index.php/en/case-studies.html)
+* **Nederlands Instituut voor Mediakunst (NIMk)** (2011), [_I/Eye_ (1993/2011, Bill Spinhoven)](https://www.scart.be/?q=en/content/case-study-report-ieye-19932011-bill-spinhoven)
+* **Emanuel Lorrain (PACKED vzw)** (c.2013), [The preservation of Danny Matthys' video works](https://www.scart.be/?q=en/content/case-study-report%C2%A0-preservation-danny-matthys-video-works)
+* **Emanuel Lorrain (PACKED vzw)** (c.2013), [_Mondophrenetic™_ (2000, Herman Asselberghs, Els Opsomer, Rony Vissers)](https://www.scart.be/?q=en/content/case-study-report-mondophrenetic%E2%84%A2-2000-herman-asselberghs-els-opsomer-rony-vissers-0)
```

Comment:
Nit: some lines above we did use a space in `c. 2013`

Reply:
I've normalised the spacing in the next commit. I prefer it as `c.2013`, and there was only one place where there was a space inbetween the `c.` and the year.

Reply:
Thanks, @retokromer ! Eagle eyes as always

Reply:
I personally prefer the other way, but as long as it is normalised I am fine with.